web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -97,6 +101,15 @@ def resolve_current_mentees(self, info, **kwargs):
     def resolve_previous_mentees(self, info, **kwargs):
         return _get_mentees(self.user, ENDED_PROGRAM_STATUS)
 
+    def resolve_confirmed_mentor_program_families(self, info, **kwargs):
+        prg = _confirmed_non_future_program_role_grant(self)
+        program_ids = latest_program_id_foreach_program_family()
+        return list(prg.filter(
+            program_role__program__pk__in=program_ids
+        ).values_list(
+            'program_role__program__program_family__name',
+            flat=True).distinct())
```

Comment:
Since the Program Affiliation section (per def of done) starts with the Home Program Family, we should probably (a) exclude the home program family from this list and (b) prepend it to the list that we return. Then your front-end code can know that the first item in the list will be their home program family, and decorate it appropriately

Reply:
> Since the Program Affiliation section (per def of done) starts with the Home Program Family, we should probably (a) exclude the home program family from this list and (b) prepend it to the list that we return. Then your front-end code can know that the first item in the list will be their home program family, and decorate it appropriately

@iankigen I notice that you gave this comment a positive reaction, but have not acted on it. Are you intending to rearrange the code as suggested? Obviously, I would prefer to see this change, because it will make the code simpler all around, but I will understand if you want to push back on it.

Reply:
@jonkiparsky I have [excluded](https://github.com/masschallenge/impact-api/blob/7be69ae8f2cb8d39e8eabe32bd0cd696604e0559/web/impact/impact/graphql/types/expert_profile_type.py#L107-L113) the home program family from the list. On the front end, I'm prepending the `home program`  to `confirmed mentor program families` list

Reply:
Gotcha, missed that (I blame github :) )